javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+const $countDown = document.querySelector('.countdown-display');
+
+const intervalID = setInterval(countDown, 1000);
+
+function countDown(num) {
+  if (parseInt($countDown.textContent) === 1) {
+    $countDown.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(intervalID);
+  } else {
+    num = parseInt($countDown.textContent);
+    num--;
+    $countDown.textContent = num;
+  }
+}
```

Comment:
Works, but we don't want to rely on the DOM for our content. It would have been better to create a `counter` variable in JS, and use that instead of getting the text from the DOM and converting it to a number